src/main/webapp/app/exam/manage/exams/exam-checklist-check/exam-checklist-check.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'jhi-exam-checklist-check',
+    templateUrl: './exam-checklist-check.component.html',
+})
+export class ExamChecklistCheckComponent implements OnInit {
+    @Input() checkAttribute = false;
+
+    constructor() {}
```

Comment:
Do we need constructor and ngOnInit if we do not do anything in them?

Reply:
Another good catch, thanks, its removed now